### PR TITLE
VIH-7964 Deserializing People and leaver to results

### PR DESCRIPTION
**Pull request by PrasannaDommalapati**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-7964

### Change description ###

- Deserialization after the schema update

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: SchedulerJobs/SchedulerJobs.Services/SchedulerJobs.Services.csproj
Comment (robertbennett1998) [inline SchedulerJobs/SchedulerJobs.Services/SchedulerJobs.Services.csproj:11]: This will need updating to not use beta once the PR merged wont it?

**Review by krishnapottekkat-hmcts — APPROVED** [review]
LGTM.

**Review by robertbennett1998 — APPROVED** [review]
LGTM. Might need to update package reference.

File: SchedulerJobs/SchedulerJobs.Services/SchedulerJobs.Services.csproj
Comment (PrasannaDommalapati) [reply SchedulerJobs/SchedulerJobs.Services/SchedulerJobs.Services.csproj:11]: We have to do it post-booking api PR approval

**Pull request merged**
